### ISSUE TITLE
Energy use last 24 hours - values disappearing/disappeared

### ISSUE THREAD

**Issue by spacecosmos** [UtilitechAS/amsreader-firmware#799]
**Describe the bug**
The data and values for **Energy use last 24 hours** are disappearing (and some already disappeared). Not sure what's going on.

**Expected behavior**
Having the data _not_ disappear.

**Screenshots**
<img width="768" alt="Screenshot 2024-05-27 at 20 49 38" src="https://github.com/UtilitechAS/amsreader-firmware/assets/55928670/e4dcd8fd-1f51-4947-9597-64a70be44be4">

**Hardware information:**
 - Country: Norway
 - Meter: Kamstrup
- Chip: esp32s2 (160MHz)
- Device: [Pow-K+]
- Version 2.3.5

**Additional context**
It's all been running fine until I saw this. Haven't updated or done anything either, except from a couple of reboots to see if it sorted itself out.

**Comment by spacecosmos** [comment]
Still a problem after multiple reboots:

<img width="1026" alt="Screenshot 2024-05-28 at 08 24 07" src="https://github.com/UtilitechAS/amsreader-firmware/assets/55928670/d0a4f4a2-1fb0-4b01-84fb-fdc33a4fd8b0">

**Comment by ArnieO** [comment]
What about the rest of your page?
Does it report power while this goes on - or has the Kamstrup Omnipower stopped sending data?

**Comment by ArnieO** [comment]
Could this be what is going on? https://www.amsleser.no/module/ets_blog/blog?id_post=33

**Comment by spacecosmos** [comment]
Sorry for the late reply. I tried to take out the device and left it disconnected for an hour hoping that would solve it - it didn't. The values display for a while just after rebooting and disappearing again. I just left everything as is an suddenly yesterday evening it started displaying correct values again. I haven't done anything and nothings really changed. So I'm a little confused.

![Screenshot_20240531_080019_Opera](https://github.com/UtilitechAS/amsreader-firmware/assets/55928670/d7d2b368-6a4b-48b3-8e0e-9b058164bd2c)

**Comment by ArnieO** [comment]
Yes, this is confusing. 
As mentioned in the article I linked, there is a firmware update available from Kamstrup for the Kamstrup Omnipower. We emailed all grid companies in Norway in March, recommending them to do the update - but did not get much response. And we know (based on reported issues from customers that we helped them solve) that some grid companies have not done the upgrade. 

So we can speculate and cross fingers - and hope that your grid company has done the upgrade now and that this is the cause and cure for what you have seen. 🤞
